sdp/processors/datasets/commoncrawl/commoncrawl.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import List
+
+import soundfile as sf
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+from sdp.processors.datasets.commoncrawl.harv_utils import split_by_vtt_new
+
+
+
+class SplitByVttSentence(BaseParallelProcessor):
```

Comment:
A couple notes for this class:
* please update docstring to be the same format that we have been using so far
* what are "proxy_fields"? could we call them something like "copy_fields"? not sure if the word "proxy" is being used correctly here. cc @karpnv